core/src/main/java/org/testcontainers/containers/GenericContainer.java

```diff
@@ -191,31 +190,10 @@ private void tryStart(Profiler profiler) {
             containerIsStarting(containerInfo);
 
             // Wait until the container is running (may not be fully started)
-            profiler.start("Wait until container state=running, or there's evidence it failed to start.");
```

Comment:
I've removed this startup attempt loop - it might have been useful back when it was added (e.g. for startup of ambassador containers), but I don't think that is the case any more. Stability seems to be unaffected by removal.

Simplifying this should make the code much more readable and easier to reason about.